examples/schema_template_bd/main.tf

```diff
@@ -49,12 +51,36 @@ resource "mso_schema_template_bd" "bd" {
   multi_destination_flooding = "flood_in_encap"
   ipv6_unknown_multicast_flooding = "optimized_flooding"
   unknown_multicast_flooding = "optimized_flooding"
-  dhcp_policy = {
+  dhcp_policies {
```

Comment:
Is is possible to add another block of dhcp policy here?

Reply:
Also, update one sample!